packages/pages/src/common/src/parsers/puckConfigParser.ts

```diff
@@ -0,0 +1,52 @@
+import fs from "node:fs";
+import SourceFileParser, { createTsMorphProject } from "./sourceFileParser.js";
+import { newConfig } from "../../../scaffold/template/sampleTemplates.js";
+import { SyntaxKind } from "ts-morph";
+
+/**
+ * ConfigParser is a class for parsing the puck config file.
+ */
+export default class ConfigParser {
```

Comment:
Any reason this a class and not just a function?

Reply:
Oh it's cuz the other parsers were classes w/ multiple functions. Made it a func.